src/foundations/theme-provider/global-styles/globalStyles.ts

```diff
@@ -10,6 +12,7 @@ import VElevation from './variables.elevation';
 import VFontFaces from './variables.font-faces';
 import VFontStyle from './variables.font-style';
 import VBorderRadius from './variables.border-radius';
+import VAnimationSpeeds from './variables.animation';
```

Comment:
mmh, i would rather name the import `VAnimation` or the file `./variables.animation-speeds`

But to prevent to many separate files and imports I would probably go with `VAnmiation`

WDYT?